packages/pixeloven-webpack/config/src/helpers/client.ts

```diff
@@ -0,0 +1,84 @@
+import {
```

Comment:
I plan on completely removing this and the server file and eventually refactoring the shared functions. I did it this way to make it easier to encapsulate logic for all the parts of the two configs so that merging them was super easy. The next phase will require using to flatten this back out.